spectrum/checks.py

```diff
@@ -576,7 +583,22 @@ def listing_of_listing(self, path):
         LOGGER.info("Loaded listing of listing %s, found links: %s", path, links)
         return links
 
+    # TODO: use elsewhere than spectrum.load?
+    def profile(self, path):
```

Comment:
could be used in `spectrum.test_journal`, but that uses sessions because of the feature flag... probably later on